src/InputTag.vue

```diff
@@ -83,7 +83,7 @@
       <span>{{ tag }}</span>
       <a v-if="!readOnly" @click.prevent.stop="remove(index)" class="remove"></a>
     </span>
-    <input v-if="!readOnly" v-bind:placeholder="placeholder" type="text" v-model="newTag" v-on:keydown.delete.stop="removeLastTag()" v-on:keydown.enter.prevent.stop="addNew(newTag)" class="new-tag"/>
+    <input v-if="!readOnly" v-bind:placeholder="placeholder" type="text" v-model="newTag" v-on:keydown.delete.stop="removeLastTag()" v-on:keydown.188.prevent.stop="addNew(newTag)" v-on:keydown.enter.prevent.stop="addNew(newTag)" class="new-tag"/>
```

Comment:
Instead of adding a new `keydown` event, I think you can chain it like this: `v-on:keydown.enter.188.prevent.stop="addNew(newTag)"`. If that works, modify it please and I'll merge it